ModelFiles/dependencies.txt

```diff
@@ -1,6 +1,6 @@
 MATLAB	9.7.0.1190202 (R2019b)
 libSBML	5.17.0
-RAVEN_toolbox	commit 1300c01
+RAVEN_toolbox	commit 6a77071
```

Comment:
please switch RAVEN back to master, as the latest version `2.4.0` is now compliant with BiGG ids. However cobratoolbox is still not fixed in master, so keep it in development :)